app/services/sign_in/token_exchanger.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+module SignIn
+  class TokenExchanger
+    attr_reader :subject_token, :subject_token_type, :actor_token, :actor_token_type, :client_id
+
+    def initialize(subject_token:, subject_token_type:, actor_token:, actor_token_type:, client_id:)
+      @subject_token = subject_token
+      @subject_token_type = subject_token_type
+      @actor_token = actor_token
+      @actor_token_type = actor_token_type
+      @client_id = client_id
+    end
+
+    def perform
+      validations
+      new_container = create_new_session
+      Rails.logger.info("[SignIn][TokenExchanger] success, new_session_handle=#{new_container.session.handle}")
```

Comment:
There's a log entry already in `generate_token_exchange_response`, do we need this one also?